2024/day2/part2.js

```diff
@@ -0,0 +1,43 @@
+import utils from '../../utils/index.js';
+
+const year = 2024;
+const day = 2;
+const part = 'B';
+let sum = 0;
+
+const input = utils.getInput();
+
+for (let line of input) {
+    const levelArray = line.split(" ");
+    let validLevel = true;
+    validLevel = validateLevel(levelArray);
+    sum += validLevel ? 1 : 0;
+}
+
+function validateLevel(levelArray, backupFlag) {
```

Comment:
backupFlag should probably have a default `backupFlag = false` so it can be called without needing to supply it.